config/rabe-fs-backup.timer

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Run RaBe Backup Service timer daily
+
+[Timer]
+# Run once per day
+OnCalendar=daily
+# Start immediately if last start was missed (ie when machine was down)
+Persistent=true
+# Delay timer randomly to spread out load (centos-7.3 and up only)
+RandomizedDelaySec=30min
+# Do not coalesce with other timers, no need to save power during a backup
+AccuracySec=1us
```

Comment:
After reading the documentation about `RandomizedDelaySec=`, I agree that the inclusion of `AccuracySec=1us` makes sense. I was a bit misguided by the power-saving argument.